x-pack/test/reporting_api_integration/reporting_and_security/spaces.ts

```diff
@@ -39,19 +39,22 @@ export default function ({ getService }: FtrProviderContext) {
     );
   };
 
-  const spacesSharedObjectsArchive =
-    'x-pack/test/functional/es_archives/reporting/ecommerce_kibana_spaces';
-
   describe('Exports and Spaces', () => {
+    const id = 'non_default_space';
     before(async () => {
-      await esArchiver.load(spacesSharedObjectsArchive); // multiple spaces with different config settings
+      await spacesService.create({ id, name: id });
+      await kibanaServer.importExport.load(
+        `x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce_kibana_${id}`,
```

Comment:
Using the space name which also happens to be part of the kbn_archive name made some sense when you were loading multiple kbn_archives into difference spaces.  But now that it's only one, I would prefer to just hard-code the whole kbn_archive name here.  The reason is, I often grep through source to find where files are being used and I wouldn't find this.